### PR TITLE
1.114.0 Changelog

### PR DESCRIPTION
**Pull request by confused-Techie**
Adds all new changelog entries.

There are still some PRs that might make it in last minute, so no worries, this won't be merged too quickly.

---

References #928

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,38 @@
 
 ## [Unreleased]
 
+## 1.114.0
+
+- Prevented an exception raised in the command palette in certain unusual filtering scenarios.
+- Fix the tag Linux binaries are uploaded to at the Rolling releases repo.
```
Comment (Daeraxa) [inline CHANGELOG.md:12]: Not trying to diminish its importance but really don't we have this section to highlight the major changes that will be seen in the 1.114 release over the 1.113 release? In this case stuff that only affects rolling release (like binary names) aren't really what people are going to be looking for in a changelog.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,38 @@
...
+- Tweak Rolling release binary upload script to reject `-dev` version strings, as it should.
```
Comment (Daeraxa) [inline CHANGELOG.md:13]: Same with this entry - not particularly relevant to 1.114 itself.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,36 @@
...
+- Fixed: CI: Fix tag Linux binaries are uploaded to for Rolling [@DeeDeeG](https://github.com/pulsar-edit/pulsar/pull/901)
+- Fixed: [command-palette] Guard against failure to highlight a match [@savetheclocktower](https://github.com/pulsar-edit/pulsar/pull/913)
+- Fixed: `symbols-view` rolling fixes [@savetheclocktower](https://github.com/pulsar-edit/pulsar/pull/861)
+- Fixed: Tree-sitter rolling fixes (February) [@savetheclocktower](https://github.com/pulsar-edit/pulsar/pull/906)
+- Updated: [meta] Update Cirrus `GITHUB_TOKEN` [@confused-Techie](https://github.com/pulsar-edit/pulsar/pull/924)
+
+### PPM
+- Fixed: Fix test failure due to missing atom command [@toddy15](https://github.com/pulsar-edit/ppm/pull/124)
+- Updated: Update syntax-variables.less to include language entity colors [@savetheclocktower](https://github.com/pulsar-edit/ppm/pull/123)
+- Added: feat(link): add --force flag [@claytoncarter](https://github.com/pulsar-edit/ppm/pull/122)
```
Comment (Daeraxa) [inline CHANGELOG.md:37]: Could these maybe promoted to the top section? I think it would be good to draw attention to the new functionality here.

Comment (confused-Techie) [reply CHANGELOG.md:37]: Just added them, it's a good point, also added some notes about Tree-Sitter fixes.

**Comment by DeeDeeG** [comment]
Maybe we can add a "highlighted change" line something like this?: "bumped dugite, should make github package work on ARM Linux"

**Review by Daeraxa — APPROVED** [review]

**Comment by Daeraxa** [comment]
Didn't notice the above comment, I think that would be good to add.

**Comment by DeeDeeG** [comment]
I can push a commit with that line in both files in a sec, if you don't get to it before me @confused-Techie.

**Comment by confused-Techie** [comment]
@DeeDeeG Just pushed the requested change.
In general I try to make everything in the changelog from earliest first order

**Review by DeeDeeG — APPROVED** [review]
Thanks for doing this, LGTM!

**Comment by confused-Techie** [comment]
Thanks for the review @Daeraxa & @DeeDeeG

**Pull request merged**